### DISCUSSION TITLE
Where do I report a bug? - Adding quasar 2 to a Vue 3 - Vite app

### DISCUSSION THREAD

**Discussion by peppermintbunny** [quasarframework/quasar#14757] [General - Components / Directives / etc]
When I use the command : vue add quasar" to a Vue3 app, in the resultant build the file quasar-variables-styl is referenced but  is missing.

Where should I report this?
And
Anyone suggest how I can handle this?  If I comment out the reference to this file then the display is messed up , and of course, the reason to use quasar is for its super components!

**Comment by suleimanas** [comment]
might be useful: https://stackoverflow.com/questions/58151768/stylus-syntax-where-do-imports-come-from

**Comment by peppermintbunny** [comment]
Many thanks!!!

From: Suleiman A. S ***@***.***>
Date: Tuesday, 1 November 2022 at 12:36
To: quasarframework/quasar ***@***.***>
Cc: peppermintbunny ***@***.***>, Author ***@***.***>
Subject: Re: [quasarframework/quasar] Where do I report a bug? - Adding quasar 2 to a Vue 3 - Vite app (Discussion #14757)

might be useful: https://stackoverflow.com/questions/58151768/stylus-syntax-where-do-imports-come-from
